theano/sandbox/rng_mrg.py

```diff
@@ -865,7 +865,10 @@ def uniform(self, size, low=0.0, high=1.0, ndim=None, dtype=None,
             nstreams = self.n_streams(size)
         rstates = self.get_substream_rstates(nstreams, dtype)
 
-        node_rstate = shared(rstates)
+        d = {}
```

Comment:
This would need some tests. Easy tests would be to just use the random generator with target=cpu for all distribution. This would at least make sure it don't crash.

Reply:
I tested with following code:
```
print rng.uniform((3,2), target='cpu')
print rng.binomial((3,2), target='cpu')
print rng.multinomial(pvals=numpy.array([[.98, .01, .01], [.01, .49, .50]]).astype('float32'), target='cpu')
print rng.choice(p=numpy.array([[.98, .01, .01], [.01, .49, .50]]).astype('float32'), replace=False, target='cpu')
print rng.multinomial_wo_replacement(pvals=numpy.array([[.98, .01, .01], [.01, .49, .50]]).astype('float32'), target='cpu')
```
no crash happened. What is a flake8 error?

Reply:
The tests should go in the one test file. For this file, the tests are in theano/sandbox/tests/tests_rng_mrg.py

Don't use print in the tests. Add a comment that you just test if it don't crash.

flake8 is a python code syntax standard. Theano enforce most of it in some tests to help make the code more readable and more consistent.

On each PR, Theano have CI (Continuous Integration) system that will run our tests automatically. In the PR, next to the commit, there if an red "x" that tell some of our tests failed. Click on it. You will see the errors.

Reply:
A fix is submitted